src/pages/discover.js

```diff
@@ -72,6 +103,19 @@ const DiscoverPage = ({ router }) => {
     });
   };
 
+  // const onLoadCollectives = (e, value) => {
+  // router.push({
+  // pathname: router.pathname,
+  // query: { ...router.query, offset: 0, show: value },
+  // });
+  // };
+
+  const collectiveChecks = {};
```

Comment:
This is not a reliable way to test for collectives properties. This `collectiveChecks` should be removed.